tests/test-timber-filters.php

```diff
@@ -1,12 +1,25 @@
 <?php
 
 class TestTimberFilters extends Timber_UnitTestCase {
-	function testRenderDataFilter() {
+
+	function testLoaderRenderDataFilter() {
 		add_filter('timber/loader/render_data', array($this, 'filter_timber_render_data'), 10, 2);
 		$output = Timber::compile('assets/output.twig', array('output' => 14) );
 		$this->assertEquals('output.twig assets/output.twig', $output);
 	}
 
+	function testRenderDataFilter() {
+		add_filter('timber/render/data', function( $data, $file ){
+			$data['post'] = array('title' => 'daaa');
+			return $data;
+		}, 10, 2);
+		ob_start();
+		Timber::render('assets/single-post.twig', array('fop' => 'wag'));
+		$str = ob_get_contents();
+		ob_end_clean();
```

Comment:
FYI, you can do `$str = ob_get_clean()` to do both buffer ops in one go.

Reply:
Thanks @acobster , applied!